src/Express.js

```diff
@@ -1,75 +1,82 @@
 'use strict';
 
+import {Router} from 'ferry';
+
 import express from 'express';
 
-class Express {
-  constructor(specification, database) {
+class ExpressAdapter extends Router {
+
+  constructor(config = {}) {
+    super(config);
     this.name = 'Express';
-    this.specification = specification;
     this.app = express();
-
-    this.initialize();
   }
 
-  route(type, model) {
+  route(action, resource) {
 
-    switch (type) {
+    switch (action) {
 
       case 'index':
         return (req, res)=> {
-          res.send('Index');
+          res.send(action);
```

Comment:
So I think from the work [here](https://github.com/ferryjs/ferry/pull/17), we will be able to pull the models out of the Ferry object that gets injected?  How will we write tests for this?